addons/sms/views/sms_registration_views.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Views should be moved into wizard directly with their matching wizards (a bit weird but that's a convention)